### PR TITLE
Bump Android NuGet dependencies versions

### PR DESCRIPTION
**Pull request by MoienTajik**
**Why?**
There are some other NuGet packages out there (In my case, `Plugin.MauiMTAdmob`) which has indirect dependency on `Xamarin.AndroidX.Fragment` version >= 1.8.3. This causes version conflict issue with `Maui.RevenueCat.InAppBilling` as the version here is lower.

### CODE REVIEW

**Comment by Kebechet** [comment]
Hello @MoienTajik 
thanks for the PR, in following days I will take a look. 
I experienced the same incompatibility issue with the exact package in the past so we will 100% merge this if there are no side-effects.

File: src/Maui.RevenueCat.Android/Maui.RevenueCat.Android.csproj
Comment (Kebechet) [inline src/Maui.RevenueCat.Android/Maui.RevenueCat.Android.csproj:26]: version is derived from the native Android library used. V`7.12.0`.
So please change it as `7.12.0.1` where 4th number is the number of patch for this version

File: src/Maui.RevenueCat.InAppBilling/Maui.RevenueCat.csproj
Comment (Kebechet) [inline src/Maui.RevenueCat.InAppBilling/Maui.RevenueCat.csproj:55]: again 7.12.0.1

File: src/Maui.RevenueCat.InAppBilling/Maui.RevenueCat.csproj
Comment (Kebechet) [inline src/Maui.RevenueCat.InAppBilling/Maui.RevenueCat.csproj:35]: please adjust this description as well

File: src/Maui.RevenueCat.Android/Maui.RevenueCat.Android.csproj
Comment (Kebechet) [inline src/Maui.RevenueCat.Android/Maui.RevenueCat.Android.csproj:27]: please adjust this description

**Review by Kebechet — CHANGES_REQUESTED** [review]
I have tested this and it seems correctly.
After you resolve those CR changes I will merge this PR and publish it.

**Comment by MoienTajik** [comment]
Done @Kebechet

**Comment by Kebechet** [comment]
Both
[Android](https://www.nuget.org/packages/Kebechet.Maui.RevenueCat.Android/7.12.0.1) and [InAppBilling](https://www.nuget.org/packages/Kebechet.Maui.RevenueCat.InAppBilling/4.5.3) nugets are released. 
Thank you for the contribution! ❤️

**Comment by MoienTajik** [comment]
Awesome! Thanks.

**Pull request merged**
